src/installs/visual-studio.ts

```diff
@@ -107,3 +113,383 @@ export function compareVersions(lhs: string, rhs: string): number {
         }
     }
 }
+
```

Comment:
Has any of this code that you're moving between files changed?  If so, would it be possible to keep the code in the old location and then do a second PR that moves it?  If that's too much trouble, can you just point me at what changed?

Reply:
The initial refactor was a direct copy, besides a rename of `kitHostTargetArch` to `getHostTargetArchString`. That was all in the first commit (3a37c61).

The only two notable changes after that were in `varsForVSInstallation`
1. factoring the `getVcVarsBatScript` logic into its own function
2. adding -vcvars_ver to the args

Reply:
Thanks. That's good to know. I have the correct diff set up in CodeFlow now.